npc/quests/seals/megingard_seal.txt

```diff
@@ -1898,6 +1898,7 @@ morocc_in,146,179,0	script	Employee#megin1	1_F_01,{
 							mes "knocks you out~^000000";
 							close2;
 							percentheal -100,0;
+							end;
```

Comment:
maybe `unitkill getcharid(3);` would be better/suitable for killing a char? 
Perhaps would be helpful in future code standardization?